### PR TITLE
feat: templator compiler

### PR DESCRIPTION
**Pull request by SonyPradana**
create cache file without excuted

### CODE REVIEW

File: tests/View/TemplatorTest.php [hunk 1]
```diff
@@ -259,4 +259,18 @@ public function itCanRenderRepeatTemplate(): void
...
+    {
+        $loader  = __DIR__ . DIRECTORY_SEPARATOR . 'sample' . DIRECTORY_SEPARATOR . 'Templators';
+        $cache   = __DIR__ . DIRECTORY_SEPARATOR . 'caches';
+
+        $view = new Templator($loader, $cache);
+        $out  = $view->compile('include.php');
+
+        $this->assertSee(trim($out), '<p>taylor</p>');
+        $this->assertFileExists($cache . DIRECTORY_SEPARATOR . md5('include.php') . '.php');
+        sleep(20);
```
Comment (anggerpradana) [inline tests/View/TemplatorTest.php]: no no no

File: src/System/View/Templator.php [hunk 2]
```diff
@@ -47,6 +47,28 @@ public function render(string $templateName, array $data, bool $cache = true): s
...
+        if (!file_exists($template_dir)) {
+            throw new ViewFileNotFound($template_dir);
+        }
+
+        $cachePath = $this->cacheDir . '/' . md5($template_name) . '.php';
+
+        $template = file_get_contents($template_dir);
+        $template = $this->templates($template);
+
+        file_put_contents($cachePath, $template);
```
Comment (anggerpradana) [inline src/System/View/Templator.php:67]: better split this (DRY)

**Review by anggerpradana — APPROVED** [review]

File: src/System/View/Templator.php (hunk 2 above)
Comment (SonyPradana) [reply src/System/View/Templator.php:67]: next pr

**Comment by SonyPradana** [comment]
ready to merge (ref #248)

**Pull request merged**